ds3-autogen-net/src/main/java/com/spectralogic/ds3autogen/net/NetHelper.java

```diff
@@ -124,6 +125,13 @@ public static String argToString(final Arguments arg) {
         }
     }
 
+    /**
+     * Determines if a list of arguments contains the specified argument
+     */
+    public static boolean containsArgument(final ImmutableList<Arguments> args, final String argName) {
```

Comment:
I am using this method from within a template, so I either need to wrap it or add Helper to the request model.